addons/account/models/account_move.py

```diff
@@ -508,7 +508,7 @@ def _sequence_fixed_regex(self):
         help="Is the move being sent asynchronously",
         compute='_compute_is_being_sent'
     )
-
```

Comment:
useless diff :smile:

Reply:
In this case, would it be better to leave the line in the code? or is there a way that wouldn't leave the diff?

Reply:
You can leave the line yess :smile: